test/features/home_page_title.feature

```diff
@@ -0,0 +1,23 @@
+@smoke
+Feature: Raketa.by tests
+
+@cart
+Scenario: Test cart
+    Given I open "https://raketa.by/"
+    When I input "скричер" into searching field
+    When I click on the first item
+    When I click Buy
+    When I go to the cart
+    When I check cost with real cost "84,90"
+
+@titles
+Scenario Outline: Page Titles <URL>
+    Given I open "<URL>"
+    Then Page title should be "<Title>"
+    When I wait "<Seconds>" seconds
```

Comment:
Why wait in the end?

Reply:
I don't know, it was in example. I can delete it, it's really unnecessary